### ISSUE TITLE
getplatertemplate.ps1 from Prior VSCode Install Is Still Executing When Open Default PowerShell Console

### ISSUE THREAD

**Issue by HJLBX** [Microsoft/vscode#46528]
Uninstalled VSCode but now every time I launch the default PowerShell, getplastertemplate.ps1 keeps executing and it is triggering the installed security software - which blocks it and creates alert notifications:

03/26/18 04:12:04 Prevented process <getplastertemplate.ps1 | C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe> from launching from <c:\users\<user>\.vscode\extensions\ms-vscode.powershell-1.6.0\modules\plaster>.

How do I stop the getplastertemplate.ps1 from exectuing every time I open the default PowerShell now ?

Visual Studio Code has been completely uninstalled and a manual cleanup of both the file system and the registry have been performed.

In other words, c:\users\<user>\.vscode\extensions\ms-vscode.powershell-1.6.0\modules\plaster does not exist.

How do I reset PowerShell to the way it was before installing VSCode ?

**Comment by dbaeumer** [comment]
@HJLBX can you please file the issue against the PowerShell extension. This repository is only about the VS Code core product.
